src/utilities/vaccineReport.js

```diff
@@ -227,12 +228,17 @@ const writeReport = async content => {
 // TODO: LOCALIZE
 const getSubject = sensor => `Temperature log report for ${sensor.name ?? sensor.macAddress}`;
 
-export const emailVaccineReport = async (sensor, user, comment, recipients) => {
+export const emailVaccineReport = async (macAddress, user, email, comment) => {
+  const sensor = UIDatabase.get('Sensor', macAddress, 'macAddress');
+
+  if (!sensor) throw new Error('Cannot find sensor');
+  if (sensor?.logs <= 0) throw new Error('No temperature logs');
+
   const content = await vaccineReport(sensor, user, comment);
   const path = await writeReport(content);
   const subject = getSubject(sensor);
```

Comment:
I'm very confused, when `sensor.name` is null (e.g. save a sensor with no name) the subject is coming up empty 😕 . Surely` sensor.name ?? sensor.macAddress` should return the macAddress? What 🧙  is going on